scripts/thanks.js

```diff
@@ -0,0 +1,33 @@
+if (!remoteUrl) {
+  var remoteUrl = "https://api.carpoolvote.com/live";
+}
+
+const handleBackendResponse = () => {
```

Comment:
ES5 – sounds like something to mention in the contributing guide. Maybe we can use jekyll to transpile to ES5 at some stage? Computers do this sort of thing better than people.